src/components/Pages/Login/LoginForm/index.js

```diff
@@ -43,22 +45,47 @@ export const LoginForm = () => {
     e.preventDefault();
     //Validate user inputs
     if (email === "" || password === "") {
-      return setError("Please fill both fields");
+      return;
```

Comment:
Here you can set the error as "required!" as a fallback, in case the disabled button fails for some reason after updating code.
You could just
`setPasswordError("required!");
setEmailError("required!");`